packages/clay-labels-input-field/src/utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * If `query` matches to `string`, returns an object with the
+ * corresponding characters. If there is no match, return null.
+ * @param {!String} query
+ * @param {!String} string
+ * @return {null|Object}
+ */
+export const match = (query, string) => {
```

Comment:
@gcmznt used [fuzzy](https://github.com/liferay/clay/pull/971/files#diff-fc5f1ca3d00e2b11a1a6029e3b277473R3). Do you think that might be an option so we don't have to code and maintain this?

Reply:
hey @jbalsas, Yeah, I thought of using it here, but I had trouble getting to the Lexicon markup result, with the fuzzy it concatenates in the characters that are match the `strong` tag, but we're having problems with the parameters that they accept HTML in Metal soy (https://github.com/metal/metal.js/issues/381). And this utility is the same fuzzy implementation, but without the option to concatenate some markup, just list which items are active... so we can get to the desired https://github.com/liferay/clay/pull/1292/files#diff-087a55086d39969ce453305b7bca6fcfR160

Reply:
Sounds good for now, then!